src/Sylius/Bundle/CoreBundle/Application/SyliusPluginTrait.php

```diff
@@ -68,6 +68,8 @@ public function getContainerExtension(): ?ExtensionInterface
         if ($this->containerExtension) {
```

Comment:
Maybe just return $this->containerExtension ?: null;

Reply:
yeah .. I guess that makes things a bit more compact and readable